suite-native/firmware/src/hooks/useIsFirmwareUpdateFeatureEnabled.tsx

```diff
@@ -0,0 +1,19 @@
+import { useSelector } from 'react-redux';
+
+import { FeatureFlag, useFeatureFlag } from '@suite-native/feature-flags';
+import {
+    selectIsFeatureEnabled,
+    Feature,
+    MessageSystemRootState,
+} from '@suite-common/message-system';
+
+export const useIsFirmwareUpdateFeatureEnabled = () => {
+    const localFeatureFlag = useFeatureFlag(FeatureFlag.IsFirmwareUpdateEnabled);
+    const remoteFeatureFlag = useSelector((state: MessageSystemRootState) =>
+        selectIsFeatureEnabled(state, Feature.firmwareUpdate, true),
+    );
+
+    const isFirmwareUpdateEnabled = localFeatureFlag && remoteFeatureFlag;
```

Comment:
This way it wouldn't be available on production for users at all, because `localFeatureFlag` has default value `isDevelopOrDebugEnv()`. And there is no way to enable it locally if it's remotely disabled. I guess OR makes more sense here.

```suggestion
    const isFirmwareUpdateEnabled = localFeatureFlag || remoteFeatureFlag;
```

Reply:
My assumption was that we will set local feature flag to `true` since we want to include it in release. If there will be OR and it will be true remote FF won't work at all.

Reply:
I see, but in that case I would propose to switch the local flag for production now. I originally assumed you'd remove the local flag completely... I'm a little worried about migration. Feature flags are persistent. So all the users of current production version now have it set to `false`.